iommi/declarative/namespace.py

```diff
@@ -12,6 +12,18 @@ def _get_type_of_namespace(dict_value):
 
 
 class Namespace(Struct):
+    """
+    Namespace represents a structure of nested dicts. It behaves like a regular
+    dictionary, with the added feature that items at nested levels can be accessed
```

Comment:
"Accessed" seems like the wrong word. The dunders are used when creating a namespace, but you can't access using them, except by using `getattr_path`, but that works on any  nested data structure.

Reply:
Thanks for the clarification! I actually hadn't realised this until you pointed it out. I have modified the docstring accordingly, hopefully it is accurate and clearer now.